src/components/slide/slide.js

```diff
@@ -293,9 +305,8 @@ export default function Slide({
             <AnimatedDiv
               ref={setStepContainer}
               onClick={handleClick}
-              tabIndex={
-                Object.entries(frameOverrideStyle).length > 0 ? 0 : undefined
-              }
+              data-in-overview-mode={inOverviewMode}
```

Comment:
Used `data-*` as a prop to prevent React Warning:
<img width="526" alt="Screen Shot 2020-12-03 at 4 56 18 PM" src="https://user-images.githubusercontent.com/16032631/101108024-7a620280-3588-11eb-8398-bf595fe04840.png">